lib/fileutils.js

```diff
@@ -43,3 +68,29 @@ function which(exe, cb){
     cb(!!exePath)
   })
 }
+
+exports.findableByWhichOrModule = findableByWhichOrModule
+function findableByWhichOrModule(cb){
+  var browser = this
+
+  findableBy(which).call(browser, function(findable){
+    if(findable){
+      cb(findable)
+    }else{
+      findableByModule.call(browser, cb)
```

Comment:
Shouldn't we look first for the module and then for the exe to find a local phantomjs, before finding the global one?